tests/test_packages.py

```diff
@@ -0,0 +1,17 @@
+import json
+
+from cf_remote.packages import Release
+
+
+def test_release():
+    with open("tests/master-releases.json") as file:
+        data = file.read()
+        json_data = json.loads(data)
+        #        print("json_data is {}".format(json_data))
+        release = Release(None, extended_data=json_data)
```

Comment:
@vpodzime here I use extended_data in the test to pass in my own test data. Thus the reason for the change in the Release class.

Reply:
I'd prefer modifying the `release` instance after its initialization instead of making the code confusing only to allow this test.